### ISSUE TITLE
macOS 10.13: Crash on start due to missing `_NSAppearanceNameDarkAqua` symbol

### ISSUE THREAD

**Issue by workflowsguy** [Komet/MediaElch#1695]
**Describe the bug**
The program crashes on lauch

**To Reproduce**
Just start the app

**Expected behavior**
Main window of program should appear

**MediaElch Version:**
 - [ ] 2.10.7-dev (nightly)
 - [x] 2.10.6 (stable), version for macOS 10.x

**Operating System:**
 - [ ] Windows
 - [x] macOS
 - [ ] Linux (distribution: _._)

**Additional context**
The top part of the crash report:
```
Process:               MediaElch [15514]
Path:                  /Applications/MediaElch.app/Contents/MacOS/MediaElch
Identifier:            com.kvibes.MediaElch
Version:               2.10.6 (2.10.6)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           MediaElch [15514]
User ID:               501

Date/Time:             2023-12-21 16:03:06.370 +0100
OS Version:            Mac OS X 10.13.6 (17G14042)
Report Version:        12
Anonymous UUID:        A41421B4-F1A6-23F0-F507-8C741C7D2643

Sleep/Wake UUID:       CB389EB8-0D90-45A4-A8F3-0008FB62F8EB

Time Awake Since Boot: 25000 seconds
Time Since Wake:       1100 seconds

System Integrity Protection: enabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x4] Symbol missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Symbol not found: _NSAppearanceNameDarkAqua
  Referenced from: /Applications/MediaElch.app/Contents/MacOS/MediaElch
  Expected in: /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit
 in /Applications/MediaElch.app/Contents/MacOS/MediaElch
```

**Comment by bugwelle** [comment]
Hi,

thank you very much for reporting it and also posting the stack trace. It could be that we accidentally require at least macOS 10.14, which is bad. :(

My best guess is: After the introduction of the [dark theme](https://successfulsoftware.net/2021/03/31/how-to-add-a-dark-theme-to-your-qt-application/), we accidentally became reliant on macOS 10.14 or later.
After a quick [Google search](https://stackoverflow.com/questions/63513229/macos-app-crashes-on-10-13-with-symbol-not-found-nsappearancenamedarkaqua), that could indeed be the case.

That, or one of our dependencies no longer supports macOS 10.13, and I didn't notice that.

Here is a version that removes the dependency. It would be awesome if you could test it. There are no changes compared to the latest stable version except the removal of auto-detection of dark system themes.

 - https://mediaelch-downloads.ameyering.de/snapshots/macOS_X/
 - https://mediaelch-downloads.ameyering.de/snapshots/macOS_X/MediaElch_macOS_Qt5_2.10.6_2023-12-22_10-04_git-no-dark-theme-802cf59d.dmg

I'm currently not at home (I will be in January again), and don't have access to my old MacBook with macOS 10.13. I can't test this fix, so it would be great to get confirmation by you. :-)


Regards,
Andre

**Comment by workflowsguy** [comment]
@bugwelle,

thanks for looking into this.

The app version you provided in the link above runs again on macOS 10.13. Cursory testing did not turn up any issues.

Cheers!

**Comment by bugwelle** [comment]
Thanks. I'll make sure to test it properly when I'm back home. After all, I don't see any reason to require macOS 10.14 for now. :)

**Comment by bugwelle** [comment]
Tested on Mac OS X 10.13: MediaElch doesn't compile due to an undefined property `effectiveAppearance`.

I can't get it to build on macOS 10.13. I had to remove the Dark-Mode related check, as it only works with macOS 10.14 or later (which I don't have).

I either have to provide a custom DMG for macOS 10.13 on top of the version for macOS 10 and 11, or just say that our Qt5 version for macOS 10 doesn't support Dark Mode anymore.  I think that's fine, since we have a separate Qt6 version for macOS 11 and later.
